src/lib/style/components/_query-wrapper-text-input.scss

```diff
@@ -2,6 +2,13 @@
 
 .QueryWrapperSearchInput,
 .QueryWrapperSqlEditorInput {
+  &__helppopoverwrapper {
+    display: grid;
```

Comment:
Is it better to use grid in this type of case?

Reply:
In this case I have one cell that I want to hardcode the width, align, and justify.  I actually initially used a flexbox, but it was too flexible ;)